.gitignore

```diff
@@ -7,6 +7,7 @@ node_modules
 package-lock.json
 
 # ocaml/reason build artifacts
+.bsb.lock
```

Comment:
shouldn't we add `**/`?

Reply:
It's sufficient like this, it matches .bsb.lock in subdirectories, too.

Reply:
I was thinking the same, but the other day, I had to make some changes for patterns below (cause it wasn't enough...). Weird, maybe I am buggy lol